package/MDAnalysis/analysis/encore/similarity.py

```diff
@@ -721,7 +720,6 @@ def hes(ensembles,
         cov_estimator="shrinkage",
         weights='mass',
         align=False,
-        details=False,
```

Comment:
removal of arguments also needs to be documented — add a `versionchanged`.

Also add a note to CHANGELOG – along the lines that hes() does not take `details` kwarg anymore and that the output is now a ...

Reply:
Remove `details` from the doc string!

Reply:
Thanks, definitely - not sure how I missed this one. 

Should I add just one `versionchanged` entry overall? Or do you mean `deprecated` for the removal of the option?

Reply:
Versionchanged states that the option was removed.

Deprecated would mean that the option is still available but should not be used. As the option is gone, it has to be a versionchanged.